insights/parsers/openshift_get_with_config.py

```diff
@@ -14,44 +14,93 @@
 
 OcGetClusterRoleBindingWithConfig - command ``oc get clusterrolebinding --config /etc/origin/master/admin.kubeconfig``
 ----------------------------------------------------------------------------------------------------------------------
-
-Examples:
-    >>> type(oc_get_cluster_role_with_config)
-    <class 'insights.parsers.openshift_get_with_config.OcGetClusterRoleWithConfig'>
-    >>> oc_get_cluster_role_with_config.role[0]
-    'admin'
-    >>> type(oc_get_clusterrolebinding_with_config)
-    <class 'insights.parsers.openshift_get_with_config.OcGetClusterRoleBindingWithConfig'>
-    >>> oc_get_clusterrolebinding_with_config.rolebinding["admin"]
-    '/admin'
 """
 
 from .. import parser, CommandParser
-from . import ParseException
+from . import ParseException, parse_fixed_table
 from insights.specs import Specs
 
 
-@parser(Specs.oc_get_cluster_role_with_config)
+@parser(Specs.oc_get_clusterrole_with_config)
 class OcGetClusterRoleWithConfig(CommandParser):
-    """Class to parse ``oc get clusterrole --config /etc/origin/master/admin.kubeconfig``"""
+    """
+    Class to parse ``oc get clusterrole --config /etc/origin/master/admin.kubeconfig``
+
+    A typical sample of the content of this file looks like::
+
+        NAME
+        admin
+        asb-access
+        asb-auth
+        basic-user
+        cluster-admin
+        cluster-debugger
+        cluster-reader
+        cluster-status
+        edit
+        hawkular-metrics-admin
+        management-infra-admin
+        namespace-viewer
+        registry-admin
+
+    Examples:
+        >>> type(oc_get_cluster_role_with_config)
+        <class 'insights.parsers.openshift_get_with_config.OcGetClusterRoleWithConfig'>
+        >>> oc_get_cluster_role_with_config.data[0]
+        'admin'
+    """
 
     def parse_content(self, content):
         if len(content) < 2 or content[0].strip() != "NAME":
-            raise ParseException("Error: ", content if content else 'empty file')
-        self.role = list(map(lambda x: x.strip(), content[1:]))
+            raise ParseException("invalid content" if content else 'empty file')
+        self.data = list(map(lambda x: x.strip(), content[1:]))
+
+    def __contains__(self, item):
```

Comment:
Since you have provided the `__contains__`, better add a `__getitems__` as well.